processing/src/main/java/org/apache/carbondata/processing/store/writer/AbstractFactDataWriter.java

```diff
@@ -317,8 +313,7 @@ public void initializeWriter() throws CarbonDataWriterException {
         .getCarbonDataFileName(fileCount, model.getCarbonDataFileAttributes().getTaskId(),
             model.getBucketId(), model.getTaskExtension(),
             "" + model.getCarbonDataFileAttributes().getFactTimeStamp());
-    this.carbonDataFileTempPath = chosenTempLocation + File.separator
-        + carbonDataFileName + CarbonCommonConstants.FILE_INPROGRESS_STATUS;
+    this.carbonDataFileTempPath = chosenTempLocation + File.separator + carbonDataFileName;
```

Comment:
For the temp location, can we add a "carbon" prefix to the path name, currently it is just a nanotime number, I think it is better if user can tell which folder is created by carbon, even it is temp folder

Reply:
ok

Reply:
ok